TCAT/Views/LiveIndicator.swift

```diff
@@ -100,6 +120,11 @@ class LiveIndicator: UIView {
         }
     }
     
+    func setColor(to color: UIColor) {
+        self.color = color
+        drawViews()
```

Comment:
wouldn't this add more layers and views on top of what's there? we should be actually changing the colors of the layer and views, not adding more

Reply:
Good call